spaconclj/spacon/resources/migrations/001-inital.up.sql

```diff
@@ -123,9 +123,7 @@ CREATE TRIGGER update_updated_at_devices
 CREATE TABLE IF NOT EXISTS public.device_locations
 (
   id SERIAL PRIMARY KEY,
-  x double precision,
-  y double precision,
-  z double precision,
+  geog geometry,
```

Comment:
Can you name the column `geom` instead?  This can cause confusion b/c PostGIS also has a [Geography](http://postgis.net/docs/geography.html) type